package NYT.NCypressClient.NProto;

import "yt/core/protos/guid.proto";
import "yt/core/protos/rpc.proto";

////////////////////////////////////////////////////////////////////////////////

message TTransactionalExt
{
    extend NRpc.NProto.TRequestHeader
    {
        optional NYT.NProto.TGuid transaction_id = 100;
    }
}

////////////////////////////////////////////////////////////////////////////////

message TAccessTrackingExt
{
    extend NRpc.NProto.TRequestHeader
    {
        optional bool suppress_access_tracking = 103;
        optional bool suppress_modification_tracking = 108;
    }
}

////////////////////////////////////////////////////////////////////////////////
